### ISSUE TITLE
SSD output is object bounding box. Is it possible to get object detection mask?

### ISSUE THREAD

**Issue by Josca** [weiliu89/caffe#151]

**Comment by weiliu89** [comment]
What do you mean by object detection mask? Won't it be straightforward to get the mask from the bbox? Or do you mean "segmentation" mask?

**Comment by Josca** [comment]
Yes, I mean "segmentation" mask. It seems to be possible f.e. as linear combination of some confolution filters, but I dont know which filters choose. Thanks for fast answer and cool object detection tool.

**Comment by weiliu89** [comment]
It doesn't have the function for producing the segmentation mask yet.

**Comment by Josca** [comment]
Ok, thanks.
